model/src/main/kotlin/ScanResult.kt

```diff
@@ -85,4 +86,19 @@ data class ScanResult(
      */
     fun filterByVcsPath(): ScanResult =
         filterByPath(provenance.vcsInfo?.takeUnless { it.type == VcsType.GIT_REPO }?.path.orEmpty())
+
+    /**
+     * Return a [ScanResult] whose [summary] contains only findings whose location / path is not matched by
+     * any entry in [ignoreFilePaths].
+     */
+    fun filterByFilePath(ignoreFilePaths: Collection<String>): ScanResult {
+        val matcher = FileMatcher(*ignoreFilePaths.toTypedArray())
```

Comment:
This constructor of `FileMatcher` converts the passed in array again to a list; so it would probably be more direct to use `FileMatcher(ignoreFilePaths.toList())`.

Reply:
Done.